stock_quant_merge/models/stock_move.py

```diff
@@ -12,4 +12,5 @@ def quants_unreserve(self):
         for move in self:
             quants = move.reserved_quant_ids
             super(StockMove, move).quants_unreserve()
-            quants.merge_stock_quants()
+            if quants and not self._context.get('disable_stock_quant_merge'):
```

Comment:
nitpicking maybe but I think `self._context` is deprecated in favor of `self.env.context`.

Otherwise LGTM

Reply:
I would be curious to know where you read that self._context was deprecated.

I had a look on odoo master:
% grep -ri "self._context.get(" *|wc -l
361
% grep -ri "self.env.context.get(" *|wc -l 
303

Reply:
@alexis-via I think @sbidoul wanted to say that using self.env.context is the good manner and the usage in OCA modules.

Reply:
Well, it's not OCA itself, but Python conventions: if you put a `_` in the beginning, this means this variable/method is internal, and it might change to unexpected values. You should use public variables instead, as they should be preserved as public API.

Reply:
@pedrobaeza @alexis-via I've found this : https://github.com/odoo/odoo/blob/1d0940d96f3f5de45b38a9ddad9af65087dd04f3/odoo/models.py#L4836

So @sbidoul was right. Use of _context is deprecated!